rosbag2_test_msgdefs/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "srv/BasicSrv.srv"
   "srv/ComplexSrvMsg.srv"
   "srv/ComplexSrvIdl.srv"
+  "action/BasicAction.action"
```

Comment:
Actually this file is not used, but the same thing goes to `.srv` test configuration.